generators/entity-server/files.js

```diff
@@ -172,16 +172,6 @@ const serverFiles = {
         },
       ],
     },
-    {
-      condition: generator => generator.searchEngine === ELASTICSEARCH && !generator.embedded && !generator.paginationNo,
-      path: SERVER_MAIN_SRC_DIR,
-      templates: [
-        {
-          file: 'package/repository/search/SortToSortBuilderListConverter.java',
```

Comment:
Can you cleanup the file here: https://github.com/jhipster/generator-jhipster/blob/main/generators/server/index.js#L457

Reply:
@maznag You will need to rebase also

Reply:
done !